ENGAGEHF/Home.swift

```diff
@@ -55,17 +59,28 @@ struct HomeView: View {
                 .tabItem {
                     Label("Education", systemImage: "brain")
                 }
+            NavigationStack {
+                DevicesTab(appName: ENGAGEHF.appName ?? "ENGAGE") {
```

Comment:
This might be more SpeziDevices-focused but I would suggest to avoid the name `Tab` in a view. Ideally, I would aim to make the view easily integratabtle in a dedicated tab, especially if the interaction with different devices and frequent pairing/unpairing is part of the core user experience.

I would therefore suggest that we rather rename this view in SpeziDevices.
A general think that I would also consider that for now, we integrate the view in the Account settings in the long run. For now, it is good to have it front and center, but I would consider that with the education, medication, home tab, and others the tab bar might get full at some point so we should be able to move it to other parts of the app if needed.